### PR TITLE
Crowdin monthly september 2020

### PR DESCRIPTION
**Pull request by Julievrz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | publish crowdin monthly september 2020
| Fixed ticket? | no

### CODE REVIEW

File: news/_posts/translation/do-you-speak-prestashop-sept-2020.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ 1. | Girdman.com (anargaragezov) | Azerbaijani | 12,291
+ 2. | Daniele1970 | Esperanto & Italian | 10,266
+ 3. | mirmal | Esperanto | 2,622
+ 4. | A. L (roynsdx) | Arabic | 1,869
+ 5. | jbonello | Maltese | 1,007
+ 6. | PrestaServicePro | Azerbaijani | 467
+ 7. | AmirRzd | Persian | 373
+ 8. | Angela Anna (Annajeronima) | Mongolian | 201
+ 9. | Tantely R. (rabtaray) | Malagasy | 191
+10. | WEB.Com.Az | Azerbaijani | 145
```
Comment (atomiix) [inline news/_posts/translation/do-you-speak-prestashop-sept-2020.md]:
```suggestion
| 1. | Girdman.com (anargaragezov) | Azerbaijani | 12,291
| 2. | Daniele1970 | Esperanto & Italian | 10,266
| 3. | mirmal | Esperanto | 2,622
| 4. | A. L (roynsdx) | Arabic | 1,869
| 5. | jbonello | Maltese | 1,007
| 6. | PrestaServicePro | Azerbaijani | 467
| 7. | AmirRzd | Persian | 373
| 8. | Angela Anna (Annajeronima) | Mongolian | 201
| 9. | Tantely R. (rabtaray) | Malagasy | 191
| 10. | WEB.Com.Az | Azerbaijani | 145
```

File: news/_posts/translation/do-you-speak-prestashop-sept-2020.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+ 1. | Girdman.com (anargaragezov) | Azerbaijani | 14,654
+ 2. | Daniele1970 | Esperanto & Italian | 6,661
+ 3. | AmirRzd | Persian | 1,505
+ 4. | Laimonas Mituzas (laimonas) | Lithuanian | 314
+ 5. | webdvl (megashopba) | Czech, Slovak | 53
+ 6. | Ronny (rbuelund) | Dannish | 49
+ 7. | Rauno Riikman (weaver) | Finnish | 47
+ 8. | Anouar Talidi (Anouar_Talidi) | Arabic | 43
+ 9. | Marcin Orzechowski (Martinovy) | Polish | 6
+ 10. | Stamatis (breezer) | Greek | 5
```
Comment (atomiix) [inline news/_posts/translation/do-you-speak-prestashop-sept-2020.md]:
```suggestion
| 1. | Girdman.com (anargaragezov) | Azerbaijani | 14,654
| 2. | Daniele1970 | Esperanto & Italian | 6,661
| 3. | AmirRzd | Persian | 1,505
| 4. | Laimonas Mituzas (laimonas) | Lithuanian | 314
| 5. | webdvl (megashopba) | Czech, Slovak | 53
| 6. | Ronny (rbuelund) | Dannish | 49
| 7. | Rauno Riikman (weaver) | Finnish | 47
| 8. | Anouar Talidi (Anouar_Talidi) | Arabic | 43
| 9. | Marcin Orzechowski (Martinovy) | Polish | 6
| 10. | Stamatis (breezer) | Greek | 5
```

Comment (Julievrz) [reply news/_posts/translation/do-you-speak-prestashop-sept-2020.md]: Thank you @atomiix !

**Review by atomiix — APPROVED** [review]

File: news/_posts/translation/do-you-speak-prestashop-sept-2020.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of September 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **13 languages**:
+ 
+| Chinese Traditional | Danish | Dutch | French |
+| **Greek** | Italian | Lithuanian | Portuguese, Brazil|
```
Comment (LouiseBonnard) [inline news/_posts/translation/do-you-speak-prestashop-sept-2020.md]:
```suggestion
| **Greek** | Italian | Lithuanian | Portuguese, Brazil |
```

**Review by LouiseBonnard — APPROVED** [review]

**Pull request merged**